content/docs/getting-started.md

```diff
@@ -18,109 +18,110 @@ redirect_from:
   - "docs/environments.html"
 ---
 
-This page is an overview of the React documentation and related resources.
+Bu sayfa React kullanım kılavuzu ve ilgili kaynaklara genel bir bakıştır.
 
-**React** is a JavaScript library for building user interfaces. Learn what React is all about on [our homepage](/) or [in the tutorial](/tutorial/tutorial.html).
+**React** kullanıcı arayüzleri oluşturabileceğiniz bir JavaScript kütüphanesidir. React'ın tam olarak ne olduğunu [ana sayfamızdan](/) veya [eğitim bölümünden](/tutorial/tutorial.html) öğrenin.
 
 ---
 
-- [Try React](#try-react)
-- [Learn React](#learn-react)
-- [Staying Informed](#staying-informed)
-- [Versioned Documentation](#versioned-documentation)
-- [Something Missing?](#something-missing)
+- [React'ı deneyin](#try-react)
+- [React'ı öğrenin](#learn-react)
+- [Haberdar olun](#staying-informed)
+- [Sürüm bilgileri](#versioned-documentation)
+- [Eksik bir şey?](#something-missing)
 
-## Try React {#try-react}
+## React'ı Deneyin {#try-react}
 
-React has been designed from the start for gradual adoption, and **you can use as little or as much React as you need.** Whether you want to get a taste of React, add some interactivity to a simple HTML page, or start a complex React-powered app, the links in this section will help you get started.
+React, aşamalı olarak benimsenmesi için en baştan tasarlandı ve  **ihtiyacınız kadar az veya daha fazla React kullanabilirsiniz** React’ın tadına bakmak, basit bir HTML sayfasına biraz etkileşim katmak veya karmaşık bir React destekli uygulama başlatmak isteyip istemediğinize göre, bu bölümdeki bağlantılar başlamanıza yardımcı olacaktır.
 
-### Online Playgrounds {#online-playgrounds}
+### Çevrimiçi Takılma Alanları {#online-playgrounds}
 
-If you're interested in playing around with React, you can use an online code playground. Try a Hello World template on [CodePen](codepen://hello-world), [CodeSandbox](https://codesandbox.io/s/new), or [Glitch](https://glitch.com/edit/#!/remix/starter-react-template).
+React ile takılmak istiyorsanız, çevrimiçi bir kod oyun alanını kullanabilirsiniz. Bir Merhaba Dünya şablonu deneyin [CodePen](codepen://hello-world), [CodeSandbox](https://codesandbox.io/s/new), veya [Glitch](https://glitch.com/edit/#!/remix/starter-react-template).
```

Comment:
`vakit gecirmek` or `oyalanmak` MAY be a better choice for `playing around`. `Takilmak` is more like `hanging out`. What do you think?